tests/test-buttercup.el

```diff
@@ -501,17 +501,40 @@
 
 (describe "The Spy "
   (let (test-function)
-    (before-each
+    (before-all
```

Comment:
Interesting. I tricked myself with the semantics there :-D

I think it'd be good to add a comment regarding this here, to prevent others from assuming a quick fix there.